test/topology_password_auth/conftest.py

```diff
@@ -0,0 +1,216 @@
+#
```

Comment:
This is almost fully copy-pasted from topology. TODO: Re-use as much code from there as I can.